README.md

```diff
@@ -6,22 +6,45 @@
 
 ## Installation
 
+### Prerequisities
+
+* GNU-Make
+* Python && pip
+* OpenJDK (required by submodule)
+* optional: Docker CE for building container images 
+
 Clone the repository:
 
 ```sh
-git clone ocrd_fileformat
-cd ocrd_fileformat
+git clone --recursive git@github.com:OCR-D/ocrd_fileformat.git
```

Comment:
Better to use HTTPS, the submodule is registered as an HTTPS URL and many firewalls don't allow outgoing SSH connections.

Reply:
Yes, I got the point.